packages/neuron-wallet/src/services/light-runner.ts

```diff
@@ -0,0 +1,148 @@
+  abstract start(): Promise<void>
+  platform(): string {
+    switch (process.platform) {
+      case 'win32':
+        return 'win'
+      case 'linux':
+        return 'linux'
+      case 'darwin':
+        return 'mac'
+      default:
+        return ''
+    }
+  }
+
+  async stop() {
+    return new Promise<void>(resolve => {
+      resetSyncTaskQueue.push(false)
+      if (this.runnerProcess) {
+        logger.info('Runner:\tkilling node')
+        this.runnerProcess.once('close', () => resolve())
+        this.runnerProcess.kill('SIGKILL')
+        this.runnerProcess = undefined
+      } else {
+        resolve()
+      }
+    })
+  }
+}
+
+export class CKBLightRunner extends NodeRunner {
+  protected networkType: NetworkType = NetworkType.Full
+
+  static getInstance(): CKBLightRunner {
+    if (!CKBLightRunner.instance) {
+      CKBLightRunner.instance = new CKBLightRunner()
+    }
+    return CKBLightRunner.instance as CKBLightRunner
+  }
+
+  protected get binary() {
+    const appPath = app.isPackaged
+      ? path.join(path.dirname(app.getAppPath()), '..', './bin')
+      : path.join(__dirname, '../../bin')
+    const binary = app.isPackaged
+      ? path.resolve(appPath, './ckb-light-client')
+      : path.resolve(appPath, `./${this.platform()}`, './ckb-light-client')
+    return this.platform() === 'win' ? binary + '.exe' : binary
+  }
+
+  private get templateConfigFile() {
+    const appPath = app.isPackaged
+      ? path.join(path.dirname(app.getAppPath()), '..', './light')
+      : path.join(__dirname, '../../light')
+    return path.resolve(appPath, './ckb_light.toml')
+  }
+
+  private get configFile() {
+    return path.resolve(SettingsService.getInstance().testnetLightDataPath, './ckb_light.toml')
+  }
+
+  initConfig() {
+    if (fs.existsSync(this.configFile)) {
+      logger.info(`CKBLightRunner:\tconfig has init, skip init...`)
+      return
+    }
+    const values = fs
+      .readFileSync(this.templateConfigFile)
+      .toString()
+      .split('\n')
+    let isStorePath = false
+    let isNetworkPath = false
+    const newValues = values.map(v => {
+      if (isStorePath) {
+        isStorePath = false
+        return `path = "${path.join(SettingsService.getInstance().testnetLightDataPath, './store')}"`
+      }
+      if (isNetworkPath) {
+        isNetworkPath = false
+        return `path = "${path.join(SettingsService.getInstance().testnetLightDataPath, './network')}"`
+      }
+      if (v === '[store]') {
+        isStorePath = true
+      } else if (v === '[network]') {
+        isNetworkPath = true
+      }
+      return v
+    })
+    if (!fs.existsSync(SettingsService.getInstance().testnetLightDataPath)) {
+      fs.mkdirSync(SettingsService.getInstance().testnetLightDataPath, { recursive: true })
+    }
+    fs.writeFileSync(this.configFile, newValues.join('\n'))
+  }
+
+  async start() {
+    if (this.runnerProcess) {
+      logger.info(`CKBLightRunner:\tckb is not closed, close it before start...`)
```

Comment:
`CKBLightRunner` could be `CKB Light Runner` for readability, and the message refer to `ckb`, maybe a typo?